{
    "algorithm_desc": "pygad_algorithm",
    "config_name": "two_way_overtaking",
    "random_seed": 1,
    "evaluation_time": null,
    "number_of_generations": 2000,
    "population_size": 5,
    "num_parents_mating": 10,
    "best_solution": null,
    "best_fitness": null,
    "mutate_eta": 5,
    "mutate_prob": 0.8,
    "crossover_eta": 10,
    "crossover_prob": 1,
    "error_message": "The number of parents to select for mating (10) cannot be greater than the number of solutions in the population (5) (i.e., num_parents_mating must always be <= sol_per_pop).\n\nTraceback (most recent call last):\n  File \"/home/fdominik98/Desktop/USVLogicSceneGeneration/src/genetic_algorithms/genetic_algorithm_base.py\", line 89, in evaluate\n    some_input = self.init_problem(self.initial_population_array[:population_size], eval_data)\n  File \"/home/fdominik98/Desktop/USVLogicSceneGeneration/src/genetic_algorithms/pygad_algorithm.py\", line 31, in init_problem\n    ga_instance = pygad.GA(\n  File \"/home/fdominik98/Desktop/USVLogicSceneGeneration/env/lib/python3.10/site-packages/pygad/pygad.py\", line 1336, in __init__\n    raise e\n  File \"/home/fdominik98/Desktop/USVLogicSceneGeneration/env/lib/python3.10/site-packages/pygad/pygad.py\", line 573, in __init__\n    raise ValueError(f\"The number of parents to select for mating ({num_parents_mating}) cannot be greater than the number of solutions in the population ({self.sol_per_pop}) (i.e., num_parents_mating must always be <= sol_per_pop).\\n\")\nValueError: The number of parents to select for mating (10) cannot be greater than the number of solutions in the population (5) (i.e., num_parents_mating must always be <= sol_per_pop).\n\n",
    "timestamp": "2024-06-20T14:27:44.326940"
}